app/labor/mailchimp_bot.rb

```diff
@@ -162,8 +162,7 @@ def md5_email(email)
   end
 
   def report_error(exception)
-    logger = Logger.new(STDOUT)
-    logger.error(exception)
+    Honeybadger.notify(exception)
```

Comment:
Question, if there is an error here is there an action that we must take? OR is this more of a "if there are a ton of errors something is wrong but 1 or 2 are normal" situation? If it is the later then we should log a metric to Datadog that we can track and alert when it hits a threshold rather than every single one.

Reply:
We don't really know but based on how often we received report/issue, I assume that it's the latter. I'll go ahead and add Datadog here.

Reply:
Oh sweet, I didnt realize you could send tags like that! I always just did strings. #TIL 

If we think occasional errors are normal then we should ONLY send it to Datadog and not to Honeybadger

Reply:
Yeah it's not exactly clear that tags could be a hash, I had to double check the [source code](https://github.com/DataDog/dogstatsd-ruby/blob/5c48caf39d50351cbbcd4184c33981c71ddd31ba/lib/datadog/statsd.rb#L102)

Let me remove the Honeybadger portion